client/src/modules/MentorRegistry/constants.ts

```diff
@@ -0,0 +1,63 @@
+export const PAGINATION = 200;
+
+export enum MentorsRegistryColumnKey {
+  Github = 'githubId',
+  Info = 'info',
+  PreferredCourses = 'preferedCourses',
+  ReceivedDate = 'receivedDate',
+  Preselected = 'preselectedCourses',
+  UpdatedDate = 'updatedDate',
+  Tech = 'technicalMentoring',
+  Languages = 'languagesMentoring',
+  StudentsLimit = 'maxStudentsLimit',
+  City = 'cityName',
+  PreferredLocation = 'preferedStudentsLocation',
+  Actions = 'actions',
+}
+
+export enum MentorsRegistryColumnName {
+  Github = 'Github',
+  Info = 'Additional',
+  PreferredCourses = 'Preferred',
+  ReceivedDate = 'Received',
+  Preselected = 'Pre-Selected',
+  UpdatedDate = 'Send',
+  Tech = 'Technologies',
+  Languages = 'Languages',
+  StudentsLimit = 'Max students',
+  City = 'City',
+  PreferredLocation = 'Students Location',
+  Actions = 'Actions',
+}
+
+export const TABS = {
+  new: [
+    MentorsRegistryColumnKey.Github,
+    MentorsRegistryColumnKey.Info,
+    MentorsRegistryColumnKey.PreferredCourses,
+    MentorsRegistryColumnKey.Preselected,
+    MentorsRegistryColumnKey.ReceivedDate,
+    MentorsRegistryColumnKey.UpdatedDate,
+    MentorsRegistryColumnKey.Tech,
+    MentorsRegistryColumnKey.Languages,
+    MentorsRegistryColumnKey.StudentsLimit,
+    MentorsRegistryColumnKey.Actions,
+  ],
+  all: [
+    MentorsRegistryColumnKey.Github,
+    MentorsRegistryColumnKey.Info,
+    MentorsRegistryColumnKey.PreferredCourses,
+    MentorsRegistryColumnKey.Preselected,
+    MentorsRegistryColumnKey.Tech,
+    MentorsRegistryColumnKey.City,
+    MentorsRegistryColumnKey.Languages,
+    MentorsRegistryColumnKey.StudentsLimit,
+    MentorsRegistryColumnKey.PreferredLocation,
+    MentorsRegistryColumnKey.Actions,
+  ],
+};
```

Comment:
You've created `MentorRegistryTabsMode` `enum`, I think we could use it here too for consistency  
```suggestion
export const TABS = {
  [MentorRegistryTabsMode.New]: [
    MentorsRegistryColumnKey.Github,
    MentorsRegistryColumnKey.Info,
    MentorsRegistryColumnKey.PreferredCourses,
    MentorsRegistryColumnKey.Preselected,
    MentorsRegistryColumnKey.ReceivedDate,
    MentorsRegistryColumnKey.UpdatedDate,
    MentorsRegistryColumnKey.Tech,
    MentorsRegistryColumnKey.Languages,
    MentorsRegistryColumnKey.StudentsLimit,
    MentorsRegistryColumnKey.Actions,
  ],
  [MentorRegistryTabsMode.All]: [
    MentorsRegistryColumnKey.Github,
    MentorsRegistryColumnKey.Info,
    MentorsRegistryColumnKey.PreferredCourses,
    MentorsRegistryColumnKey.Preselected,
    MentorsRegistryColumnKey.Tech,
    MentorsRegistryColumnKey.City,
    MentorsRegistryColumnKey.Languages,
    MentorsRegistryColumnKey.StudentsLimit,
    MentorsRegistryColumnKey.PreferredLocation,
    MentorsRegistryColumnKey.Actions,
  ],
};
```

Reply:
fixed